### DISCUSSION TITLE
Question for the Payment Gateway Implementation

### DISCUSSION THREAD

**Discussion by tinc-x** [luanlabs/fluxity-interface#62] [Q&A]
Hi team,

I noticed that the payment gateway functionality doesn’t seem to be implemented yet in the Wagent project. Is this correct?

If so, as a blockchain developer, I’d be interested in collaborating to help develop this feature. Let me know if there’s an opportunity to contribute!

Looking forward to your response.

**Comment by ceydae** [comment]
The project works fine. Thanks

**Comment by plural-pinelabs** [comment]
Hi, You may use any payment gateway services using the API integration approach.
